library/std/src/sys/stdio/trusty.rs

```diff
@@ -58,20 +76,43 @@ pub fn panic_output() -> Option<impl io::Write> {
     Some(Stderr)
 }
 
-fn _write(fd: i32, message: &[u8]) -> io::Result<usize> {
-    let mut iov = libc::iovec { iov_base: message.as_ptr() as *mut _, iov_len: message.len() };
+// FIXME: This should be in libc with a proper value. Several platforms use
```

Comment:
Afaik Trusty does not have a limit for this (yet).

Reply:
Perhaps not a constant that's exposed to users, but is there a limit in the implementation? If so, I want to make sure to avoid issues like found in https://github.com/rust-lang/rust/pull/138879.

Reply:
Not that I know of, but 1024 should be safe to use. The [kernel code](https://cs.android.com/android/platform/superproject/main/+/main:trusty/kernel/lib/trusty/iovec.c;l=143) implements an infinite iterator over the user iovec, and avoids copying the whole array.

Reply:
In that case, I'll change it to `libc::c_int::MAX`, since that's what the API limits it to. We don't need to limit it to something arbitrarily lower. Windows does it like that, for example.